PropLineTool_v1/PropLineTool/PropLineTool.cs

```diff
@@ -1678,49 +1389,39 @@ private float DefaultSpacingFenceMode()
             /// Calculates all item positions for all PLT modes and all control modes.
             /// </summary>
             /// <returns></returns>
-            private bool CalculateAllPositions(bool continueDrawing)
-            {
+            private bool CalculateAllPositions(bool continueDrawing) {
                 bool _result = false;
 
                 //moved from CalculateAll()
                 float _initialOffset = 0f;
                 Vector3 _lastFenceEndPoint = Vector3.down;
 
-                if (continueDrawing)
-                {
+                if (continueDrawing) {
                     _initialOffset = segmentState.lastFinalOffset;
                     _lastFenceEndPoint = segmentState.lastFenceEndpoint;
-                }
-                else
-                {
-                    switch (drawMode)
-                    {
+                } else {
+                    switch (drawMode) {
```

Comment:
@Alterran Do we even need this switch statement? It's always going to run the body of the case, regardless of what `drawMode` is?